Assets/Code/Read/SourceStudioModel.cs

```diff
@@ -990,7 +980,7 @@ short[] ReadSkinFamilies(BinaryReader br,int offs,int count, int refCount)
 
 		public void ParseVvd(string name)
 		{
-
+            name = name.Replace(" ", "");
```

Comment:
Why?

Reply:
i definitely remember this causing a few issues with loading